apps/front/pages/trip.vue

```diff
@@ -0,0 +1,326 @@
+<template>
+  <div v-if="!loading" class="relative w-screen h-screen">
+    <TripMap class="absolute z-0" :routes="routes" />
+    <div
+      class="absolute bottom-0 lg:top-0 left-0 right-0 lg:w-1/3 p-2 lg:p-12 lg:m-6 z-10 flex flex-col"
+      :class="{ 'rounded-l': drawerOpen, rounded: !drawerOpen }"
+      style="background-color: rgba(255, 255, 255, 0.9)"
+    >
+      <div class="sticky top-0 flex flex-col items-center lg:text-xl text-sm z-20 mb-2 text-center">
+        <UButton
+          class="md:hidden lg:hidden"
+          variant="ghost"
+          icon="i-heroicons-chevron-double-down"
+          @click="openClose"
+        ></UButton>
+        <div class="mb-6">
+          <p>
+            <span class="font-bold"> {{ t('trip.recap.1') }} </span>
+            {{ t('trip.recap.2') }}
+            <span class="font-bold text-primary">
+              {{ $route.query.departure }}
+            </span>
+            {{ t('trip.recap.3') }}
+            <span class="font-bold text-primary">{{ $route.query.arrival }}</span>
+          </p>
+          <p>
+            {{ t('trip.recap.4') }}
+            <span class="text-primary font-bold">{{
+              new Date($route.query.d_start).toLocaleDateString('fr-FR', {
+                day: 'numeric',
+                month: 'short',
+                year: 'numeric',
+              })
+            }}</span>
+            {{ t('trip.recap.5') }}
+            <span class="text-primary font-bold">{{
+              new Date($route.query.d_end).toLocaleDateString('fr-FR', {
+                day: 'numeric',
+                month: 'short',
+                year: 'numeric',
+              })
+            }}</span>
+          </p>
+        </div>
+        <UTabs v-if="!collasped" :items="items" />
+      </div>
+      <transition
```

Comment:
```suggestion
      <Transition
```
Vue / Nuxt already built in components should be capitalized (good practices)